controllers/infra/drivers/aws/create_instance.go

```diff
@@ -142,11 +142,11 @@ func (d Driver) createInstances(hosts *v1.Hosts, infra *v1.Infra) error {
 			Value: &value,
 		},
 	)
-	if infra.Spec.SSH.PkName == "" {
-		if err := d.createKeyPair(infra); err != nil {
-			return err
-		}
-	}
+	//if infra.Spec.SSH.PkName == "" {
+	//	if err := d.CreateKeyPair(infra); err != nil {
+	//		return err
+	//	}
+	//}
```

Comment:
If not used anymore, just delete this code block

Reply:
ok